src/2023/styles/styles.css

```diff
@@ -231,3 +231,25 @@ ul {
 .bold {
   font-weight: bold;
 }
+
+.social {
+  margin: var(--space-l);
+}
+
+.social--networks {
+  display: flex;
+  gap: var(--space-l);
+}
+
+.social--networks a {
+  opacity: 0.5;
+  color: white;
```

Comment:
What do you think about adding this color to the theme and using it as a variable?

Reply:
Yup 👍